__tests__/seed.test.js

```diff
@@ -90,7 +90,8 @@ describe('GET /api/articles/:article_id', () => {
                         body: 'I find this existence challenging',
                         created_at: "2020-07-09T20:11:00.000Z",
                         votes: 100,
-                        article_img_url: 'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700'
+                        article_img_url: 'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700',
+                        comment_count: "11"
```

Comment:
When using TDD, we prefer leaving old tests alone, and instead prefer writing new tests when our endpoints get more functionality. So instead of tweaking this to add an extra comment_count, it's preferable to make a new test that checks just for the comment count.